Makefile

```diff
@@ -1,9 +1,59 @@
+.PHONY: test test-run test-teardown sec
```

Comment:
lint and fmt are bringing in changes in a lot of file. This would be taken up as separate PR to help with review.